draft-howard-rats-coserv.md

```diff
@@ -97,26 +102,78 @@ The terminology from CBOR {{-cbor}}, CDDL {{-cddl}} and COSE {{-cose}} applies;
 in particular, CBOR diagnostic notation is defined in {{Section 8 of -cbor}}
 and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.
 
-# CoSERV Query Language
+# CoSERV Information Model {#secinfomodel}
 
-The CoSERV query language enables Verifiers to specify the desired characteristics of Endorsements and Reference Values based on the environment in which they are applicable.
-This section presents the CBOR data model for CoSERV queries.
+## Overview
+
+CoSERV is designed to facilitate query-response transactions between a producer and a consumer.
+In the RATS model, the producer is either an Endorser or a Reference Value Provider, and the consumer is a Verifier.
+CoSERV defines a single top-level data type that can be used for both queries and result sets.
+Queries are authored by the consumer (Verifier), while result sets are authored by the producer (Endorser or Reference Value Provider) in response to the query.
+A CoSERV data object always contains a query.
+When CoSERV is used to express a result set, the query is retained alongside the result set that was yielded by that query.
+This allows consumers to verify a match between the query that was sent to the producer, and the query that was subsequently returned with the result set.
+Such verification is useful because it mitigates against security threats arising from any untrusted infrastructure or intermediaries that might reside between the producer and the consumer.
```

Comment:
Non-native speaker question: is “mitigate against something” or “mitigate something”?  (In Italian, “mitigate” does not require any proposition because it is a transitive verb.)

Reply:
Good catch. Slapdash usage on my part. Will correct.